osu.Game.Rulesets.Taiko/Mods/TaikoModFlashlight.cs

```diff
@@ -79,7 +79,11 @@ protected override void Update()
 
                     flashlightProperties.Validate();
                 }
+                else
+                {
+                    UpdateFlashlightSize(GetSize());
```

Comment:
At least this is in the correct neighbourhood of a proper fix now, but it's still wrong. It makes `flashlightProperties` entirely useless, nullifying the optimisations from the `LayoutValue`.

The actual problem here is that this logic regressed in a recent pull. I'll describe in more detail in the issue thread.